feat(curriculum): add registration form workshop to cert (#56724)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/workshop-registration-form/60f1922fcbd2410527b3bd89.md

---
id: 60f1922fcbd2410527b3bd89
title: Step 3
challengeType: 0
dashedName: step-3
---

# --description--

In the previous lecture videos, you learned how to work with viewport units.

> Remember that the `vh` unit stands for viewport height, and is equal to 1% of the `height` of the viewport. This makes it relative to the viewport height.

Reviewer: ```suggestion
Remember, the `vh` unit stands for viewport height and represents 1% of the viewport's total height. This makes it relative to the viewport's height.
```
